src/intro-to-c/operators/operators-intro.md

```diff
@@ -0,0 +1,288 @@
+- **`>`** : greater than. If the left value is greater than the right value, it returns 1.
+- **`<`** : less than. If the left value is less than the right value, it returns 1.
+- **`>=`** : greater than or equal to. If the left value is greater than or equal to the right value, it returns 1.
+- **`<=`** : less than or equal to. If the left value is less than or equal to the right value, it returns 1.
+
+Let's take a look at a coding example:
+
+```c
+#include <stdio.h>
+
+int main(void)
+{
+    int a = 3, b = 3, c = 7;
+
+    printf("%d == %d is %d \n", a, b, a == b);
+    printf("%d == %d is %d \n", a, c, a == c);
+    printf("%d != %d is %d \n", a, b, a != b);
+    printf("%d != %d is %d \n", a, c, a != c);
+    printf("%d > %d is %d \n", a, b, a > b);
+    printf("%d > %d is %d \n", a, c, a > c);
+    printf("%d < %d is %d \n", a, b, a < b);
+    printf("%d < %d is %d \n", a, c, a < c);
+    printf("%d >= %d is %d \n", a, b, a >= b);
+    printf("%d >= %d is %d \n", a, c, a >= c);
+    printf("%d <= %d is %d \n", a, b, a <= b);
+    printf("%d <= %d is %d \n", a, c, a <= c);
+
+    return 0;
+}
+```
+
+The output is:
+```
+3 == 3 is 1 
+3 == 7 is 0 
+3 != 3 is 0 
+3 != 7 is 1 
+3 > 3 is 0 
+3 > 7 is 0 
+3 < 3 is 0 
+3 < 7 is 1 
+3 >= 3 is 1 
+3 >= 7 is 0 
+3 <= 3 is 1 
+3 <= 7 is 1 
+```
+Remember: 1 is true, 0 is false.
+
+## Logical Operators
+A logical operator connects two or more expressions and returns either 0 or 1 depending upon whether the compound expression results true or false. 
+
+- **`&&`** : Logical AND, it is true only if all operands are true
+- **`||`** : Logical OR, it is true if either one operand is true
+- **`!`** : Logical NOT, it is used to reverse the logical state. If a condition is true, then NOT will make it false.
+
+Let's look at an example:
+```c
+#include <stdio.h>
+
+int main(void)
+{
+    int a = 3, b = 3, c = 7, result;
+
+    result = (a == b) && (c > b);
+    printf("(a == b) && (c > b) is %d \n", result);
+
+    result = (a == b) && (c < b);
+    printf("(a == b) && (c < b) is %d \n", result);
+
+    result = (a == b) || (c < b);
+    printf("(a == b) || (c < b) is %d \n", result);
+
+    result = (a != b) || (c < b);
+    printf("(a != b) || (c < b) is %d \n", result);
+
+    result = !(a != b);
+    printf("!(a != b) is %d \n", result);
+
+    result = !(a == b);
+    printf("!(a == b) is %d \n", result);
+
+    return 0;
+}
+```
+The output is
+```
+(a == b) && (c > b) is 1 // both expressions are true, so the compound expression is true
+(a == b) && (c < b) is 0 // the first expression is true but the second is not, so the compound expression is false
+(a == b) || (c < b) is 1 // the first expression is true, so the compound expression is true
+(a != b) || (c < b) is 0 // both expressions are false, so the compound expression is false
+!(a != b) is 1 // a != b will return 0 but since we have the !, then !0 is 1
+!(a == b) is 0 // a == b will return 1 but since we have the !, then !1 = 0
```

Comment:
The comments are really good but I think it makes this look cramped, maybe align the comments so they're a little easier to read

```suggestion
(a == b) && (c > b) is 1     // both expressions are true, so the compound expression is true
(a == b) && (c < b) is 0     // the first expression is true but the second is not, so the compound expression is false
(a == b) || (c < b) is 1     // the first expression is true, so the compound expression is true
(a != b) || (c < b) is 0     // both expressions are false, so the compound expression is false
!(a != b) is 1               // a != b will return 0 but since we have the !, then !0 is 1
!(a == b) is 0               // a == b will return 1 but since we have the !, then !1 = 0
```